src/ui/components/summary/Summary.tsx

```diff
@@ -28,7 +28,7 @@ const markDownCitations = (summary: string) => {
 
       return accum;
     }, [] as string[])
-    .join(" ");
+    .join("");
```

Comment:
@ofermend Because this change touches the `ui` directory, it should go into https://github.com/vectara/vectara-ui first. Otherwise we risk overwriting changes to the `ui` directory the next time we sync it with the Vectara UI repo.

Reply:
Would you mind making a PR in the Vectara UI repo to mirror this change?

Reply:
Yes good point. PR added to vectara-ui